R/data.R

```diff
@@ -781,3 +781,32 @@
 #'   \item{time_point}{Smaple timepoint.}
 #' }
 "gambl_metadata"
+
+
+#' Mirage Metrics
+#'
+#' A data frame with Quality Control metrics for a collection of GAMBL samples.
+#' This resource is from the supplemental table of PMID: 36219881.
+#'
+#' @format ## `mirage_metrics`
+#' A data frame with 2376 rows and 17 columns
+#' \describe{
+#'   \item{sample_id}{Sample identifier.}
+#'   \item{AverageBaseQuality}{Average base quality.}
+#'   \item{AverageInsertSize}{Average insert size.}
+#'   \item{AverageReadLength}{Average read length.}
+#'   \item{PairsOnDiffCHR}{}
+#'   \item{TotalReads}{Total reads.}
+#'   \item{TotallyUniquelyMapped}{Total uniquely mapped reads.}
+#'   \item{TotalUnmappedreads}{Total unmapped reads.}
+#'   \item{TotalDuplicatedreads}{Total duplicated reads.}
+#'   \item{ProportionReadsDuplicated}{Proportion of duplicated reads.}
+#'   \item{ProportionReadsMapped}{Proportion of mapepd reads.}
+#'   \item{MeanCorrectedCoverage}{Mean corrected coverage.}
+#'   \item{ProportionTargetsNoCoverage}{Proportion of targets with no coverage.}
+#'   \item{ProportionCoverage10x}{Proportion of reads with at least 10x coverage.}
```

Comment:
This and the 30x should be `Proportion of genome or target space with at least 10X coverage`